frameworks/elastic/universe/resource.json

```diff
@@ -6,8 +6,8 @@
       "bootstrap-zip": "{{artifact-dir}}/bootstrap.zip",
       "scheduler-zip": "{{artifact-dir}}/elastic-scheduler.zip",
       "elasticsearch-jre-tar-gz": "https://downloads.mesosphere.com/elastic/assets/openjdk-{{elasticsearch-openjdk-version}}_linux-x64_bin.tar.gz",
-      "elasticsearch-tar-gz": "https://downloads.mesosphere.com/elastic/assets/elasticsearch-{{elastic-version}}-linux-x86_64.tar.gz",
-      "diagnostics-zip": "https://downloads.mesosphere.com/elastic/assets/support-diagnostics-{{support-diagnostics-version}}-dist.zip",
+      "elasticsearch-tar-gz": "https://s3.amazonaws.com/downloads.mesosphere.io/elastic/assets/elasticsearch-{{elastic-version}}-linux-x86_64.tar.gz",
+      "diagnostics-zip": "https://s3.amazonaws.com/downloads.mesosphere.io/elastic/assets/support-diagnostics-{{support-diagnostics-version}}-dist.zip",
```

Comment:
almost all the tests passed (assuming 1.12 test was flake) so let's use official `downloads.mesosphere.com` URIs

Reply:
Updated